products/BellHybrid/apps/application-bell-onboarding/windows/OnBoardingFinalizeWindow.cpp

```diff
@@ -6,6 +6,11 @@
 #include <gui/widgets/Icon.hpp>
 #include <i18n/i18n.hpp>
 
+namespace
+{
+    constexpr unsigned int popup_icon_y_alignment = 22U;
+} // namespace
+
```

Comment:
Couldn't it use constxpr from `StyleCommon.hpp`?

Reply:
Done